src/ocean/Assets.ts

```diff
@@ -338,11 +343,12 @@ export class Assets extends Instantiable {
    */
   public async search(text: string): Promise<QueryResult> {
     return this.ocean.metadataCache.queryMetadata({
-      text,
       page: 1,
       offset: 100,
       query: {
-        value: 1
+        query_string: {
+          query: text
+        }
       },
       sort: {
         value: 1
```

Comment:
Shouldn't this be `price.value: 1` if it point to the price value? if it's something else please ignore this comment 😃 . If it point to price value than we need to change also line 240 accordingly.